en/Extending Obsidian/Obsidian URI.md

```diff
@@ -21,140 +30,239 @@ The `action` parameter is the action that you would like to perform. Available a
 - `daily` to create or open your daily note.
 - `search` to open a search.
 
-
 > [!warning] Encoding
 > Ensure that your values are properly URI encoded. For example, forward slash characters `/` must be encoded as `%2F` and space characters must be encoded as `%20`.
-> 
- This is especially important because an improperly encoded "reserved" character may break the interpretation of the URI. [See here for details](https://en.wikipedia.org/wiki/Percent-encoding).
+>
+> This is especially important because an improperly encoded "reserved" character may break the interpretation of the URI. [See here for details](https://en.wikipedia.org/wiki/Percent-encoding).
+
+## Get an Obsidian URL
+
+To get an Obsidian URL for any note, use the **Copy Obsidian URL** command. This command is available in:
+
+- The [[Command palette]]
+- The three-dot menu in the note header
+- The right-click context menu for files in the file explorer
+
+The command copies an `obsidian://open` URL to your clipboard that points to the note.
```

Comment:
I find this vague at initial read. I believe  the URI isn't _exactly_ `obsidian://open`, but is `obsidian://open?STUFF_HERE_SO_IT_CAN_GET_TO_THE_NOTE`. I suppose that's what's meant with "an `prefix` URL", but I feel like this could be better somehow.